packages/core/src/common/_mixins.scss

```diff
@@ -103,6 +103,39 @@ $pt-dark-intent-text-colors: (
   @include force-hardware-acceleration();
 }
 
+// A collection of mixins that utilize the CSS Flexbox model to standardize
+// the layout and composition of Blueprint components.
+
+// pt-flex-parent styles
+@mixin pt-flex-parent() {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@mixin pt-flex-parent-fill() {
+  width: 100%;
+}
+
+@mixin pt-flex-parent-direction-horizontal {
+  flex-direction: row;
+}
+
+@mixin pt-flex-parent-direction-vertical {
+  flex-direction: column;
+}
+
+// pt-flex-child styles
+@mixin pt-flex-child {
+  flex: 0 0 auto;
+}
+
+@mixin pt-flex-child-fill {
+  flex: 1 1 auto;
+  min-width: 0;
+  min-height: 0;
```

Comment:
why clear the min sizes? what if I want min sizes?

Reply:
Should this be added individually by flex-children themselves? If you look at the docs, the `.pt-input` (somehow, browser default?) has `min-width: 146px` by default. If you shrink the `.pt-input-group` significantly, you'd need to overwrite its `min-width`.

https://6488-71939872-gh.circle-artifacts.com/0/home/circleci/project/packages/docs-app/dist/index.html#core/components/forms/input-group